packages/ui/routes/groups/_groupId/index.vue

```diff
@@ -137,29 +152,59 @@ export default {
       <bs-group-menu />
     </template>
     <client-only>
-      <v-tabs ref="tabs" :value="`group-${tab}`" centered>
+      <v-tabs ref="tabs" :value="activeTab" centered>
         <v-tabs-slider color="accent" />
-        <v-tab href="#group-informations">
+        <v-tab
+          href="#group-informations"
+          @click="activeTab = 'group-informations'"
```

Comment:
why did you add @click handlers on all tabs ? Does it fix something ?

Reply:
because before when we refresh the page we cant return to the last clicked tab. and we always return to the first tab and we cant change the url manually using the href, so to fix the problem i added the `activeTab`

Reply:
- The `href` attribute provides the connection between each `v-tab` and its corresponding `v-tab-item`.
- The `@click` event updates the `activeTab` value, which is used to control which tab is currently active.